From 1a0998ff81aae523544b6f0a2afd71ea8667a5db Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 23 Jul 2015 15:58:07 +0200
Subject: [PATCH] Change ipv6 check to warning instead of fatal error

The compile server might not have ipv6 but that does not prevent us from
building conky and run it on a machine which has ipv6.
---
 cmake/ConkyPlatformChecks.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/ConkyPlatformChecks.cmake b/cmake/ConkyPlatformChecks.cmake
index a2ebafc..26d0bea 100644
--- a/cmake/ConkyPlatformChecks.cmake
+++ b/cmake/ConkyPlatformChecks.cmake
@@ -105,7 +105,7 @@ endif(BUILD_IRC)
 if(BUILD_IPV6)
 	find_file(IF_INET6 if_inet6 PATHS /proc/net)
 	if(NOT IF_INET6)
-		message(FATAL_ERROR "/proc/net/if_inet6 unavailable")
+		message(WARNING "/proc/net/if_inet6 unavailable")
 	endif(NOT IF_INET6)
 endif(BUILD_IPV6)
 
-- 
2.4.6

